### ISSUE TITLE
Question: how to model the 'sellers' at auctions?

### ISSUE THREAD

**Issue by sdevalk** [linked-art/linked.art#493]
Hi all,

A question:

I'm modeling a dataset with auctions and using the [Auctions pattern](https://linked.art/model/provenance/auctions/#auction) for this.

My dataset also contains the names of the persons that sold their artworks at these auctions. However, it's unknown which artworks were sold, and to whom - all I know is that these persons participated in these events in the role of sellers. How can I model this with Linked Art?

I could use `crm:P11_had_participant` [1]. But this is rather vague - it doesn't make clear that these persons were sellers. (And: what if my dataset, at some point, is also going to contain the names of buyers - how could I model these, and distinguish them from the sellers?)

Or could I use a generic variation of the [Purchase of Lot](https://linked.art/model/provenance/auctions/#purchase-of-lot) pattern, by saying that an auction consists of one or more `crm:E8_Acquisition`, each containing a `crm:P23_transferred_title_from` (without a `crm:P22_transferred_title_to`)?

Thanks!

[1] Similar to https://github.com/linked-art/linked.art/issues/453

**Comment by Habennin** [comment]
Instantiate an instance of E7 Activity and then use the role of pattern which is

P9 -> E7 ->p2 -> E55 Type "Seller/Selling"
P9 -> E7 ->p14-> E39 seller goes here

The root E7 is your auction event which you can tag with the appropriate aat auction event vocab from AAT.

**Comment by sdevalk** [comment]
@Habennin Thank you - that makes sense!

In Turtle this would look like:

```turtle
<https://.../some-auction>
  a crm:E7_Activity ;
  crm:P2_has_type <http://vocab.getty.edu/aat/300054751> ; # Auction
  crm:P9_consists_of [
    a crm:E7_Activity ;
    crm:P2_has_type <http://vocab.getty.edu/aat/300078131> ; # Selling
    crm:P14_carried_out_by <https://.../some-person>
  ] .
```

**Comment by Habennin** [comment]
Hi @sdevalk , you're welcome.

Yes, precisely. In practice, I end up using the direct p14 under an activity very little because

a) users tend to want to know in what capacity some person / institution acted and
b) users tend to want to find information of the same kind in the same place

so it becomes rather arbitrary to say 'well if its what you consider to be the main actor look under E7 p14 E39 but if its an actor with a role then look under E7 p9 E7 p14 E39 etc.

So I just end up putting everything under the above pattern and it is both semantically correct and consistent for the data consumer.

**Comment by sdevalk** [comment]
@Habennin Thanks again!

If I understand you correctly you do not commonly use a pattern like:

```turtle
<https://.../some-auction>
  a crm:E7_Activity ;
  crm:P2_has_type <http://vocab.getty.edu/aat/300054751> ; # "Auction event"
  crm:P14_carried_out_by <https://.../some-actor> .
```

But instead something like:

```turtle
<https://.../some-auction>
  a crm:E7_Activity ;
  crm:P2_has_type <http://vocab.getty.edu/aat/300054751> ; # "Auction event"
  crm:P9_consists_of [
    a crm:E7_Activity ;
    crm:P2_has_type <http://vocab.getty.edu/aat/300419998> ; # E.g. "Auctioning"?
    crm:P14_carried_out_by <https://.../some-actor> ;
  ] .
```

**Comment by azaroth42** [comment]
Right. It's the same "role" pattern as for role in Production: https://linked.art/model/object/production/#multiple-artists 
Use `part_of` to create an aspect of the overall activity, classify it as to the role, and associate the actor with `carried_out_by`.

**Comment by sdevalk** [comment]
@azaroth42 Thanks, Rob!

**Comment by azaroth42** [comment]
Came to write it up and it gets trickier with the API. There would be both parts of the auction (seller's role) and part_of of the auction (the auction of the lots). This seems like a recipe for utter confusion. I think we should discuss options.

**Comment by beaudet** [comment]
Would this work?  The existing "offered at auction" term seems to be helpful in this case.
```

{
  "@context": "https://linked.art/ns/v1/linked-art.json",
  "id": "https://example.org/event/auction-1965-05-01",
  "type": "Activity",
  "classified_as": [
    {
      "id": "http://vocab.getty.edu/aat/300054766",
      "type": "Type",
      "label": "auction"
    }
  ],
  "label": "Sotheby’s Auction, May 1965",
  "timespan": {
    "type": "TimeSpan",
    "begin_of_the_begin": "1965-05-01",
    "end_of_the_end": "1965-05-01"
  },
  "carried_out_by": [
    {
      "id": "https://viaf.org/viaf/280786875",
      "type": "Group",
      "label": "Sotheby's"
    }
  ],
  "part": [
    {
      "type": "Activity",
      "classified_as": [
        {
          "id": "http://vocab.getty.edu/aat/300420001",
          "type": "Type",
          "label": "offered at auction"
        }
      ],
      "carried_out_by": {
        "id": "http://vocab.getty.edu/ulan/500004138",
        "type": "Person",
        "label": "J. Paul Getty"
      }
    }
  ]
}
```

**Comment by azaroth42** [comment]
Coming back to this... the selling activity is part_of the auction? We just flip `part` in the example above to `part_of` on a new record.

No?

**Comment by azaroth42** [comment]
Example in the wild already:
https://provenance.production.getty.fargeo.com/research/provenance/report/1c7a0b86-96a2-3def-a3e3-7af4820254a2
